### DISCUSSION TITLE
Please tell me about your favourite Movie

### DISCUSSION THREAD

**Discussion by carlos-paniagua** [NAIST-SE/AttractingContributors#885] [General]
## What is your favourite Movie?
Please tell us about any genre.
If it's difficult, you can also tell us about a film that impressed you recently!

https://github.com/carlos-paniagua/AttractingContributors-Movies

## あなたの好きな映画はなんですか？
ジャンルを問わず教えてください
難しければ最近印象に残った映画でもOKです！

https://github.com/carlos-paniagua/AttractingContributors-Movies

**Comment by ALORAMFER** [comment]
あなたのリストにnorwegian woodを追加するプルリクエストを送りました。
私のところにもプルリクエストしてもらえないでしょうか？あなたにとっての最高のサッカーチームを追加してほしいです。
https://github.com/ALORAMFER/AttractingContributors-SoccerTeams

I made a pull request to add "norwegian wood" to your list.
Could you add your favorite soocer team to the list at the link below?
https://github.com/ALORAMFER/AttractingContributors-SoccerTeams

**Comment by k-mizoguch1** [comment]
plz
https://github.com/kzu000/Food-what-do-you-like

**Comment by Gloria109** [comment]
My favorite movie is Pretty Woman. This film has an interesting plot, and the actors' performances are awesome
